packages/components/tabs/src/tab-group.ts

```diff
@@ -43,48 +67,156 @@ export class TabGroup extends ScopedElementsMixin(LitElement) {
     attributeOldValue: true
   };
 
+  /** All slotted tabs. */
+  #allTabs: Tab[] = [];
+
+  /** Whether more button needs to be shown */
+  #showMore = false;
+
+  /** Button used to show all tabs */
+  #moreButton!: HTMLButtonElement;
+
+  /** Observe the tablist width. */
+  #sizeObserver = new ResizeObserver(() => {
+    requestAnimationFrame(() => this.#updateSelectionIndicator());
+  });
+
   /** The slotted tabs. */
   @queryAssignedElements({ slot: 'tabs' }) tabs?: Tab[];
 
   /** The current tab node selected in the tab group. */
   @state() private selectedTab: Tab | null = this.#initialSelectedTab;
 
+  /** The current tab node selected in the tab listbox (dropdown). */
+  @state() private selectedTabInListbox: Tab | null = this.#initialSelectedTab;
+
+  /** Emits when the tab has been selected/changed. */
   @event() tabChange!: EventEmitter<number>;
 
   /** Renders the tabs vertically instead of the default horizontal  */
   @property({ reflect: true }) vertical = false;
 
+  /** The alignment of tabs inside sl-tab-group  */
+  @property({ reflect: true }) alignment: TabsAlignment = 'left';
+
+  /** The listbox element with all tabs list. */
+  @query('[popover]') listbox!: HTMLElement;
+
   /** Get the selected tab button, or the first tab button. */
   get #initialSelectedTab(): Tab | null {
     return this.querySelector('sl-tab[selected]') || this.querySelector('sl-tab:not([disabled])');
   }
 
   override render(): TemplateResult {
+    this.#moreButton = this.shadowRoot?.querySelector('#more-btn') as HTMLButtonElement;
+    this.#allTabs = Array.from(this.querySelectorAll('sl-tab'))?.map(tab => tab.cloneNode(true)) as Tab[];
+
     return html`
-      <div @click=${this.#handleTabChange} role="tablist" @keydown=${this.#handleKeydown} part="tab-list">
-        <span class="indicator" role="presentation"></span>
-        <slot name="tabs" @slotchange=${() => this.#rovingTabindexController.clearElementCache()}></slot>
+      <div class="container" part="container" @keydown=${this.#handleKeydown}>
+        <div class="wrapper" part="wrapper">
+          <div @click=${this.#handleTabChange} role="tablist" part="tab-list">
+            <span class="indicator" role="presentation"></span>
+            <slot name="tabs" @slotchange=${() => this.#rovingTabindexController.clearElementCache()}></slot>
+            <div
+              id="tabs-popover"
+              ${anchor({ element: this.#moreButton, position: this.vertical ? 'bottom-start' : 'bottom-end' })}
```

Comment:
Since you use the `anchor` directive here, do we still need to set `position: fixed` in the CSS if popover isn't supported?

Reply:
Without  `position: fixed;` it's not working properly in Firefox:

<img width="531" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/8ed754da-5274-453c-aacf-0c44d06df3ab">